hypervisor/Makefile

```diff
@@ -394,6 +394,7 @@ ifneq ($(BOARD_FILE)$(SCENARIO_FILE),)
 	@cat $(HV_OBJDIR)/.cfg_src_result
 	@if [ "`sed -n /successfully/p $(HV_OBJDIR)/.cfg_src_result`" = "" ]; then exit 1;	fi
 	@echo "Import hypervisor configurations from Config-xmls, configurations in source code are ignored!"
+	@sed -i 's/#define HV_CONFIG_TOOL ""/#define HV_CONFIG_TOOL " with acrn-config"/g' $(VERSION)
```

Comment:
According to this line, the string will be added is " with acrn-config", not just "acrn-config", right?
Please align with the commit message.

Reply:
sure, will update in commit info.